active_storage-send_zip.gemspec

```
@@ -41,5 +41,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest', '~> 5.0'
   spec.add_development_dependency 'rake'
   spec.add_dependency 'rails', '> 5.2'
-  spec.add_dependency 'rubyzip', '< 3.0'
+  spec.add_dependency 'rubyzip', '< 4.0'
```

```suggestion
  spec.add_dependency 'rubyzip', ['>= 2.1.0', '< 4.0']
```

`create: true` arg was only supported/added in [2.1.0](https://github.com/rubyzip/rubyzip/releases/tag/v2.1.0) via https://github.com/rubyzip/rubyzip/pull/418